governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md

```diff
@@ -0,0 +1,110 @@
+
+As a pioneer in the DeFi space with billions in funds in the protocol, the Maker ecosystem is an attractive target for blackhat hackers who look to steal funds or disrupt the protocol. With millions lost regularly in the cryptocurrency space, it is likely that the Maker protocol, its key people, and its key infrastructure, are targeted on a regular basis.
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP39/MIP39c2-Subproposals/MIP39c2-SP24.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP39c2SP24)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp24-adding-immunefi-security-core-unit-is-001/10810)
+
+### MIP40c3-SP42
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+MIP40c3-SP42 adds the budget for Core Unit IS-001: Immunefi Security. It contains:
+
+* Total Budget Cap: The hard limit voted on by Governance
+* First-month forecast: The actual first month estimated expenses
+* Regular monthly forecast
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP40/MIP40c3-Subproposals/MIP40c3-SP42.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP42)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp42-adding-immunefi-security-core-unit-budget-is-001/10813)
+
+### MIP40c3-SP41
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+MIP40c3-SP41 adds the MKR Incentive Plan budget for Core Unit IS-001: Immunefi Security. It contains:
+
+* Total MKR Expenditure Cap
+* Estimated MKR Expenditure (based on the current team)
+* Escrow Wallet mechanism
+
+The IS Core Unit follows the Sustainable Ecosystem Scaling Core Unit (SES-001) proposal for MKR Vesting.
+
+MKR incentives have been determined based on the Program discussed here. This is a 3-year vesting plan with 1-year cliff vest.
+
+The IS core unit has been working under the incubation program of the SES since October 2021. The members of this core unit will take that as the start date for the vesting schedule.
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP40/MIP40c3-Subproposals/MIP40c3-SP41.md)
```

Comment:
Link should probably be:
https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP40/MIP40c3-Subproposals/MIP40c3-SP41.md